actionsList.ts

```diff
@@ -0,0 +1,68 @@
+interface IToDo {
+    name: string,
+    date: Date,
+    finished: boolean,
+}
+
+class Action implements IToDo {
+    constructor(public name: string, public date: Date, public finished: boolean) {
+    }
+
+    public finishTask() {
```

Comment:
Acest comportament ar trebui sa fie declarat pe ActionList, nu pe Action. Adica, Actionlist este cel care ar trebui sa poata termina un Action in functie de ceva criteriu, nume, index, etc.